### PR TITLE
Add support for cold audio latency testing

### PR DESCRIPTION
**Pull request by BenjaminFair**

### CODE REVIEW

**Comment by BenjaminFair** [comment]
This is still kind of experimental and may not be the most accurate (especially for input), but I wanted to try to get it merged before the end of my internship.

**Comment by kamrik** [comment]
I'm likely to be away from any device where i can reasonably read the code
till im back on 11th. If you need to build something on top, just go ahead
and merge it, we can always fix stuff later.

**Comment by kamrik** [comment]
Looks good. Merging.

File: arduino/walt/walt.ino [hunk 1]
```diff
@@ -39,6 +39,7 @@
 
 #define CMD_AUDIO               'A'
 #define CMD_BEEP                'B'
+#define CMD_BEEP_STOP           'S'
```
Comment (gkasten) [inline arduino/walt/walt.ino]: It would be good if there was a single common source for the .java and .ino (.c) definitions

**Comment by gkasten** [comment]
It would be good if there was a single common source for the .java and .ino (.c) definitions of CMD_*

**Comment by kamrik** [comment]
That's doable but pretty heavy. One way I can think of is to link the ino file into the Anroid app tree and add it as a resource and then parse it in Java as text file. Maybe there are some libs for this. But then we loos the convenience of auto-completion in the IDE and compiler checks (though, this is minor).

**Pull request merged**